helm_deploy/hmpps-community-accommodation-tier-2-ui/values.yaml

```diff
@@ -56,9 +56,8 @@ generic-service:
       REDIS_HOST: "primary_endpoint_address"
       REDIS_AUTH_TOKEN: "auth_token"
 
-  allowlist:
-    groups:
-      - internal
```

Comment:
hmm, I think we probably *do* want to apply IP filtering to all environments other than production? i.e. it makes sense for our internal development and qa environments to be limited to that `internal` group?

Reply:
I had the same thought but didn't want to deviate from current implementation, which actually sets `null` on **all** environments. I think it would make sense, to have the internal group for all but prod. If there's consensus I'm happy to update this, does this need confirming that there are no need for users from `prison` group to access dev/test/pre-prod? I.e. for user research, debugging, etc?

Reply:
I think `internal` will cover all CAS2 users of dev/test/pre-prod but you should maybe check with the product owner? If, for example, certain prison staff needed to use pre-prod then you could add the `prisons` group from `hmpps-ip-allowlists` to the preprod helm values. But if NACRO needed to use pre-prod then you'd need to remove all IP controls from pre-prod. But I don't believe that's currently the case.